openedx/adg/admin_dashboard/applications/views.py

```diff
@@ -101,6 +104,36 @@ def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
         self.application_type = kwargs.get('application_type')
 
+    def get(self, request, *args, **kwargs):
+        """
+        Attaches csv report for download with response.
+
+        Returns:
+            HttpResponse: Response with grades report.
+        """
+        if not request.GET.get('download_report') or request.GET.get('download_report') == 'False':
+            return super().get(request, *args, **kwargs)
+
+        applications = self.get_queryset()
+        export_email_range = request.GET.get('export_email_range')
+
+        if export_email_range != 'all':
+            from_day, to_day = get_dates_from_duration(export_email_range, **request.GET.dict())
+            if not from_day or not to_day:
+                return super().get(request, *args, **kwargs)
+
+            applications = applications.filter(
+                user__application_hub__submission_date__gte=from_day, user__application_hub__submission_date__lte=to_day
+            )
+
+        emails = list(applications.values_list('user__email'))
+        emails.insert(0, ('Emails',))
+
+        response = get_http_response_with_csv_file_attachment(
+            emails, file_name=f'accepted_applicants_{now().date()}.csv'
```

Comment:
Let's create a constant for filename naming and then import and format that constant here with the current date.